internal/event/client.go

```diff
@@ -14,6 +15,18 @@ import (
 
 const source = "opg.poas.makeregister"
 
+var events = map[any]string{
+	(*UidRequested)(nil):               "uid-requested",
```

Comment:
This is cool, I've wanted to have a map of types in the past but could never work out how to do it 🙌